ui/index.d.ts

```diff
@@ -116,6 +116,11 @@ export interface ITreeItemPickerContext extends IActionContext {
      */
     suppressCreatePick?: boolean;
 
+    /**
+     * If set to true, the quick pick dialog will not close when focus moves out.
```

Comment:
This should say "defaults to `true`" at the end

Reply:
Thank you, Eric! Fixed the comment.

It's best to have this released soon, so I've updated the version to `0.35.2`. Thanks a lot.